collect_app/src/androidTest/java/org/odk/collect/android/support/pages/NotificationDrawer.kt

```diff
@@ -154,4 +151,17 @@ class NotificationDrawer {
             device
         }
     }
+
+    /**
+     * It appears that sometimes the notification drawer does not close automatically when it should
+     * such as after clicking on a notification or its action. This could be due to a bug in Android.
+     */
+    private fun closeNotificationDrawerIfOpened() {
+        val device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+        device.wait(Until.hasObject(By.text("Manage")), TimeInMs.ONE_SECOND)
```

Comment:
Sorry I probably wasn't very clear in my last comment. I meant we want to wait for "Manage" to be gone, not for it to appear. Then if that wait fails we can do `device.pressBack`. That way we account for any delay in closing the drawer. In more abstract terms, we want to "wait for the notification drawer to close and then close it ourselves if it's not closed".

Reply:
Ah ok, done.